v2/index/index.go

```diff
@@ -102,6 +102,13 @@ func GetFirst(idx Index, key cid.Cid) (uint64, error) {
 	return firstOffset, err
 }
 
+// func Has(idx Index, key cid.Cid) (bool, error) {
+// 	has := false
+// 	err := idx.GetAll(key, func(_ uint64) bool {
+//
+// 	})
+// }
+
```

Comment:
It seems the scope of this PR expanded since my last comment. Is that intended?

Reply:
my fingers decided to type `git commit -a` after I carefully selected my changes ...

Reply:
no worries thank you